2015-10-12 00:00:12.985 [info] <0.388.0> Application lager started on node 'core@alex-N550JK'
2015-10-12 00:00:13.063 [warning] <0.387.0> lager_error_logger_h dropped 56 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-12 00:00:13.065 [info] <0.388.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-12 00:00:13.065 [info] <0.388.0> Application ibot_events started on node 'core@alex-N550JK'
2015-10-12 00:00:20.009 [info] <0.388.0> Application mnesia exited with reason: stopped
2015-10-12 00:00:20.104 [info] <0.388.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-12 04:18:21.364 [info] <0.388.0> Application lager started on node 'core@alex-N550JK'
2015-10-12 04:18:28.405 [warning] <0.387.0> lager_error_logger_h dropped 57 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-12 04:18:28.405 [info] <0.388.0> Application mnesia exited with reason: stopped
2015-10-12 04:18:28.478 [info] <0.388.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-12 04:18:58.618 [error] <0.688.0> gen_server ibot_events_srv_node_interaction terminated with reason: bad return value: ok
2015-10-12 04:18:58.618 [error] <0.688.0> CRASH REPORT Process ibot_events_srv_node_interaction with 0 neighbours exited with reason: bad return value: ok in gen_server:terminate/7 line 804
2015-10-12 04:18:58.619 [error] <0.686.0> Supervisor ibot_events_sup had child ibot_events_srv_node_interaction started with ibot_events_srv_node_interaction:start_link() at <0.688.0> exit with reason bad return value: ok in context child_terminated
2015-10-12 04:20:58.292 [info] <0.388.0> Application lager started on node 'core@alex-N550JK'
2015-10-12 04:21:05.353 [warning] <0.387.0> lager_error_logger_h dropped 61 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-12 04:21:05.353 [info] <0.388.0> Application mnesia exited with reason: stopped
2015-10-12 04:21:05.439 [info] <0.388.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-12 04:43:33.829 [info] <0.388.0> Application lager started on node 'core@alex-N550JK'
2015-10-12 04:43:40.904 [warning] <0.387.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-12 04:43:40.904 [info] <0.388.0> Application mnesia exited with reason: stopped
2015-10-12 04:43:40.986 [info] <0.388.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-12 04:46:11.474 [info] <0.388.0> Application lager started on node 'core@alex-N550JK'
2015-10-12 04:46:18.537 [warning] <0.387.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-12 04:46:18.537 [info] <0.388.0> Application mnesia exited with reason: stopped
2015-10-12 04:46:18.611 [info] <0.388.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-12 05:09:25.490 [info] <0.388.0> Application lager started on node 'core@alex-N550JK'
2015-10-12 05:09:32.548 [warning] <0.387.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-12 05:09:32.548 [info] <0.388.0> Application mnesia exited with reason: stopped
2015-10-12 05:09:32.630 [info] <0.388.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-12 05:23:16.734 [info] <0.388.0> Application lager started on node 'core@alex-N550JK'
2015-10-12 05:23:23.729 [warning] <0.387.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-12 05:23:23.729 [info] <0.388.0> Application mnesia exited with reason: stopped
2015-10-12 05:23:23.803 [info] <0.388.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-12 05:40:13.152 [info] <0.388.0> Application lager started on node 'core@alex-N550JK'
2015-10-12 05:40:20.189 [warning] <0.387.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-12 05:40:20.189 [info] <0.388.0> Application mnesia exited with reason: stopped
2015-10-12 05:40:20.282 [info] <0.388.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-13 02:45:17.683 [info] <0.388.0> Application lager started on node 'core@alex-N550JK'
2015-10-13 02:45:24.709 [warning] <0.387.0> lager_error_logger_h dropped 60 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-13 02:45:24.709 [info] <0.388.0> Application mnesia exited with reason: stopped
2015-10-13 02:45:24.781 [info] <0.388.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-13 02:51:42.763 [info] <0.388.0> Application lager started on node 'core@alex-N550JK'
2015-10-13 02:51:49.803 [warning] <0.387.0> lager_error_logger_h dropped 64 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-13 02:51:49.803 [info] <0.388.0> Application mnesia exited with reason: stopped
2015-10-13 02:51:49.908 [info] <0.388.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-13 02:54:50.105 [info] <0.388.0> Application lager started on node 'core_test@alex-N550JK'
2015-10-13 02:54:57.102 [warning] <0.387.0> lager_error_logger_h dropped 62 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-13 02:54:57.102 [info] <0.388.0> Application mnesia exited with reason: stopped
2015-10-13 02:54:57.176 [info] <0.388.0> Application mnesia started on node 'core_test@alex-N550JK'
2015-10-13 02:55:43.009 [info] <0.388.0> Application lager started on node 'core@alex-N550JK'
2015-10-13 02:55:50.046 [warning] <0.387.0> lager_error_logger_h dropped 60 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-13 02:55:50.046 [info] <0.388.0> Application mnesia exited with reason: stopped
2015-10-13 02:55:50.145 [info] <0.388.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-15 23:22:46.998 [info] <0.388.0> Application lager started on node 'core@alex-N550JK'
2015-10-15 23:22:47.000 [warning] <0.387.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-15 23:22:47.069 [info] <0.388.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-15 23:22:47.069 [info] <0.388.0> Application ibot_events started on node 'core@alex-N550JK'
2015-10-15 23:22:54.024 [info] <0.388.0> Application mnesia exited with reason: stopped
2015-10-15 23:22:54.106 [info] <0.388.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-19 04:13:03.062 [info] <0.388.0> Application lager started on node 'core@alex-N550JK'
2015-10-19 04:13:10.111 [warning] <0.387.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-19 04:13:10.111 [info] <0.388.0> Application mnesia exited with reason: stopped
2015-10-19 04:13:10.193 [info] <0.388.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-19 04:24:17.435 [info] <0.388.0> Application lager started on node 'core@alex-N550JK'
2015-10-19 04:24:24.463 [warning] <0.387.0> lager_error_logger_h dropped 64 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-19 04:24:24.464 [info] <0.388.0> Application mnesia exited with reason: stopped
2015-10-19 04:24:24.564 [info] <0.388.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-20 00:20:44.332 [info] <0.388.0> Application lager started on node 'core@alex-N550JK'
2015-10-20 00:20:51.388 [warning] <0.387.0> lager_error_logger_h dropped 60 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-20 00:20:51.388 [info] <0.388.0> Application mnesia exited with reason: stopped
2015-10-20 00:20:51.510 [info] <0.388.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-20 20:37:32.759 [info] <0.388.0> Application lager started on node 'core@alex-N550JK'
2015-10-20 20:37:39.859 [warning] <0.387.0> lager_error_logger_h dropped 65 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-20 20:37:39.860 [info] <0.388.0> Application mnesia exited with reason: stopped
2015-10-20 20:37:39.936 [info] <0.388.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-23 03:33:16.587 [info] <0.388.0> Application lager started on node 'core@alex-N550JK'
2015-10-23 03:33:23.632 [warning] <0.387.0> lager_error_logger_h dropped 63 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-23 03:33:23.632 [info] <0.388.0> Application mnesia exited with reason: stopped
2015-10-23 03:33:23.720 [info] <0.388.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-24 03:31:35.228 [info] <0.388.0> Application lager started on node 'core@alex-N550JK'
2015-10-24 03:31:42.262 [warning] <0.387.0> lager_error_logger_h dropped 57 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-24 03:31:42.262 [info] <0.388.0> Application mnesia exited with reason: stopped
2015-10-24 03:31:42.336 [info] <0.388.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-26 02:48:54.928 [info] <0.388.0> Application lager started on node 'core@alex-N550JK'
2015-10-26 02:48:55.056 [warning] <0.387.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-26 02:48:55.056 [info] <0.388.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-26 02:48:55.057 [info] <0.388.0> Application ibot_events started on node 'core@alex-N550JK'
2015-10-26 02:49:01.980 [info] <0.388.0> Application mnesia exited with reason: stopped
2015-10-26 02:49:02.051 [info] <0.388.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-28 04:36:11.480 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-28 04:36:18.511 [warning] <0.388.0> lager_error_logger_h dropped 64 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-28 04:36:18.511 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-28 04:36:18.610 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-28 04:40:29.482 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-28 04:40:36.522 [warning] <0.388.0> lager_error_logger_h dropped 56 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-28 04:40:36.522 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-28 04:40:36.615 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-28 04:47:07.862 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-28 04:47:14.892 [warning] <0.388.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-28 04:47:14.892 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-28 04:47:14.965 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-28 05:07:16.588 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-28 05:07:23.662 [warning] <0.388.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-28 05:07:23.662 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-28 05:07:23.742 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-28 05:11:03.402 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-28 05:11:10.440 [warning] <0.388.0> lager_error_logger_h dropped 61 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-28 05:11:10.440 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-28 05:11:10.523 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-28 05:14:26.652 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-28 05:14:33.706 [warning] <0.388.0> lager_error_logger_h dropped 57 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-28 05:14:33.706 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-28 05:14:33.794 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-28 05:17:20.045 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-28 05:17:27.074 [warning] <0.388.0> lager_error_logger_h dropped 41 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-28 05:17:27.074 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-28 05:17:27.163 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-28 05:20:54.984 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-28 05:20:55.057 [warning] <0.388.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-28 05:20:55.058 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-28 05:20:55.058 [info] <0.389.0> Application ibot_events started on node 'core@alex-N550JK'
2015-10-28 05:21:02.020 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-28 05:21:02.114 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-29 00:36:51.885 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-29 00:36:58.926 [warning] <0.388.0> lager_error_logger_h dropped 60 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-29 00:36:58.926 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-29 00:36:59.006 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-29 00:38:42.713 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-29 00:38:49.783 [warning] <0.388.0> lager_error_logger_h dropped 57 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-29 00:38:49.783 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-29 00:38:49.876 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-29 00:42:38.890 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-29 00:42:45.950 [warning] <0.388.0> lager_error_logger_h dropped 63 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-29 00:42:45.950 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-29 00:42:46.030 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-29 01:29:12.072 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-29 01:29:19.116 [warning] <0.388.0> lager_error_logger_h dropped 55 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-29 01:29:19.116 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-29 01:29:19.199 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-29 01:32:33.372 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-29 01:32:40.417 [warning] <0.388.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-29 01:32:40.417 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-29 01:32:40.496 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-29 01:34:37.913 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-29 01:34:44.986 [warning] <0.388.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-29 01:34:44.986 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-29 01:34:45.057 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-29 02:37:10.944 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-29 02:37:11.013 [warning] <0.388.0> lager_error_logger_h dropped 57 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-29 02:37:11.014 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-29 02:37:11.014 [info] <0.389.0> Application ibot_events started on node 'core@alex-N550JK'
2015-10-29 02:37:18.564 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-29 02:37:18.659 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-29 03:26:31.651 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-29 03:26:38.678 [warning] <0.388.0> lager_error_logger_h dropped 57 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-29 03:26:38.679 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-29 03:26:38.770 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-29 03:30:18.312 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-29 03:30:25.345 [warning] <0.388.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-29 03:30:25.345 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-29 03:30:25.446 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-29 04:02:03.034 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-29 04:02:10.111 [warning] <0.388.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-29 04:02:10.112 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-29 04:02:10.204 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-29 04:04:30.844 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-29 04:04:37.907 [warning] <0.388.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-29 04:04:37.907 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-29 04:04:37.978 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-29 04:07:47.003 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-29 04:07:54.021 [warning] <0.388.0> lager_error_logger_h dropped 14 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-29 04:07:54.021 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-29 04:07:54.084 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-29 04:46:09.442 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-29 04:46:16.470 [warning] <0.388.0> lager_error_logger_h dropped 65 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-29 04:46:16.470 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-29 04:46:16.553 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-29 04:55:49.995 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-29 04:55:50.068 [warning] <0.388.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-29 04:55:50.070 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-29 04:55:50.071 [info] <0.389.0> Application ibot_events started on node 'core@alex-N550JK'
2015-10-29 04:55:57.026 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-29 04:55:57.109 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-29 04:58:51.586 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-29 04:58:58.610 [warning] <0.388.0> lager_error_logger_h dropped 64 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-29 04:58:58.610 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-29 04:58:58.634 [error] emulator Error in process <0.702.0> on node 'core@alex-N550JK' with exit value: {[{reason,{case_clause,{aborted,{node_not_running,'core@alex-N550JK'}}}},{mfa,{ibot_webi_handler,websocket_handle,3}},{stacktrace,[{ibot_db_func,get_from_mnesia,2,[{file,"src/ibot_db_func.erl"},{line,51}]},{ibot_db_func_config... 


2015-10-29 04:58:58.635 [error] <0.543.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.702.0> exit with reason: {[{reason,{case_clause,{aborted,{node_not_running,'core@alex-N550JK'}}}},{mfa,{ibot_webi_handler,websocket_handle,3}},{stacktrace,[{ibot_db_func,get_from_mnesia,2,[{file,"src/ibot_db_func.erl"},{line,51}]},{ibot_db_func_config,get_node_info,1,[{file,"src/ibot_db_func_config.erl"},{line,117}]},{ibot_db_func_config,generate_nodes_info_to_list,2,[{file,"src/ibot_db_func_config.erl"},{line,138}]},{ibot_core_app,get_project_node_from_config,0,[{file,"src/ibot_core_app.erl"},{line,116}]},{ibot_webi_handler,websocket_handle,3,[{file,"src/ibot_webi_handler.erl"},{line,78}]},{cowboy_websocket,handler_call,7,[{file,"src/cowboy_websocket.erl"},{line,598}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,529}]}]},{msg,{text,<<"{\"getNodes\" : \"\"}">>}},{req,[{socket,#Port<0.931>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.702.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},58740}},{host,<<"localhost">>},{host_info,undefined},{port,10100},{path,<<"/websocket">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"host">>,<<"localhost:10100">>},{<<"connection">>,<<"Upgrade">>},{<<"pragma">>,<<"no-cache">>},{<<"cache-control">>,<<"no-cache">>},{<<"upgrade">>,<<"websocket">>},{<<"origin">>,<<"http://localhost:10100">>},{<<"sec-websocket-version">>,<<"13">>},{<<"user-agent">>,<<"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/39.0.2171.65 Chrome/39.0.2171.65 Safari/537.36">>},{<<"accept-encoding">>,<<"gzip, deflate, sdch">>},{<<"accept-language">>,<<"en-US,en;q=0.8">>},{<<"sec-websocket-key">>,<<"Z0iktxWRHvZTSKO1EDhgLQ==">>},{<<"sec-websocket-extensions">>,<<"permessage-deflate; client_max_window_bits">>}]},{p_headers,[{<<"sec-websocket-extensions">>,[{<<"permessage-deflate">>,[<<"client_max_window_bits">>]}]},{<<"upgrade">>,[<<"websocket">>]},{<<"connection">>,[<<"upgrade">>]}]},{cookies,undefined},{meta,[{websocket_compress,false},{websocket_version,13}]},{body_state,waiting},{multipart,undefined},{buffer,<<>>},{resp_compress,false},{resp_state,done},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined_state}],[{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,529}]}]}

2015-10-29 04:58:58.770 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-29 05:00:51.416 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-29 05:00:58.472 [warning] <0.388.0> lager_error_logger_h dropped 56 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-29 05:00:58.472 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-29 05:00:58.552 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-30 00:56:50.167 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-30 00:56:57.216 [warning] <0.388.0> lager_error_logger_h dropped 65 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-30 00:56:57.216 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-30 00:56:57.321 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-30 01:30:18.179 [error] emulator Error in process <0.831.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-10-30 01:32:39.421 [error] emulator Error in process <0.834.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-10-30 01:35:27.403 [error] emulator Error in process <0.842.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-10-30 01:40:51.353 [error] <0.652.0> gen_server ibot_nodes_srv_service terminated with reason: bad argument in call to erlang:list_to_atom(new_cpp_service) in ibot_db_func_services:register_server_service/3 line 54
2015-10-30 01:40:51.353 [error] <0.652.0> CRASH REPORT Process ibot_nodes_srv_service with 0 neighbours exited with reason: bad argument in call to erlang:list_to_atom(new_cpp_service) in ibot_db_func_services:register_server_service/3 line 54
2015-10-30 01:40:51.353 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_service started with ibot_nodes_srv_service:start_link() at <0.652.0> exit with reason bad argument in call to erlang:list_to_atom(new_cpp_service) in ibot_db_func_services:register_server_service/3 line 54 in context child_terminated
2015-10-30 01:43:01.449 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-30 01:43:08.504 [warning] <0.388.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-30 01:43:08.504 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-30 01:43:08.584 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-30 02:11:44.755 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-30 02:11:52.001 [warning] <0.388.0> lager_error_logger_h dropped 64 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-30 02:11:52.001 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-30 02:11:52.083 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-30 03:48:55.902 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-30 03:49:02.922 [warning] <0.388.0> lager_error_logger_h dropped 63 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-30 03:49:02.922 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-30 03:49:03.015 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-30 04:50:21.300 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-30 04:50:28.341 [warning] <0.388.0> lager_error_logger_h dropped 55 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-30 04:50:28.341 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-30 04:50:28.439 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-30 04:50:44.169 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-30 04:50:51.230 [warning] <0.388.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-30 04:50:51.230 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-30 04:50:51.314 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-30 04:57:33.154 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-30 04:57:40.182 [warning] <0.388.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-30 04:57:40.182 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-30 04:57:40.285 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-10-30 04:58:10.694 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-10-30 04:58:17.737 [warning] <0.388.0> lager_error_logger_h dropped 64 messages in the last second that exceeded the limit of 50 messages/sec
2015-10-30 04:58:17.737 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-10-30 04:58:17.821 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 04:40:30.031 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 04:40:37.060 [warning] <0.388.0> lager_error_logger_h dropped 64 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 04:40:37.060 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 04:40:37.145 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 04:41:26.065 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 04:41:33.108 [warning] <0.388.0> lager_error_logger_h dropped 57 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 04:41:33.108 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 04:41:33.192 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 04:41:49.761 [error] <0.651.0> gen_server ibot_nodes_srv_connector terminated with reason: {enoent,[{erlang,open_port,[{spawn_executable,false},[{line,1000},stderr_to_stdout,{args,["/home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp.py","first_cpp","alex-N550JK","core@alex-N550JK","ibot_nodes_srv_connector","ibot_nodes_srv_topic","ibot_nodes_srv_service","ibot_nodes_srv_ui_interaction","ibot_events_srv_node_interaction",jv]}]],[]},{ibot_nodes_srv_connector,run_node,1,[{file,"src/ibot_nodes_srv_connector.erl"},{line,245}]},{ibot_nodes_srv_connector,handle_cast,2,...},...]}
2015-11-01 04:41:49.762 [error] <0.651.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: {enoent,[{erlang,open_port,[{spawn_executable,false},[{line,1000},stderr_to_stdout,{args,["/home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp.py","first_cpp","alex-N550JK","core@alex-N550JK","ibot_nodes_srv_connector","ibot_nodes_srv_topic","ibot_nodes_srv_service","ibot_nodes_srv_ui_interaction","ibot_events_srv_node_interaction",jv]}]],[]},{ibot_nodes_srv_connector,run_node,1,[{file,"src/ibot_nodes_srv_connector.erl"},{line,245}]},{ibot_nodes_srv_connector,handle_cast,2,...},...]} in gen_server:terminate/7 line 804
2015-11-01 04:41:49.763 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.651.0> exit with reason {enoent,[{erlang,open_port,[{spawn_executable,false},[{line,1000},stderr_to_stdout,{args,["/home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp.py","first_cpp","alex-N550JK","core@alex-N550JK","ibot_nodes_srv_connector","ibot_nodes_srv_topic","ibot_nodes_srv_service","ibot_nodes_srv_ui_interaction","ibot_events_srv_node_interaction",jv]}]],[]},{ibot_nodes_srv_connector,run_node,1,[{file,"src/ibot_nodes_srv_connector.erl"},{line,245}]},{ibot_nodes_srv_connector,handle_cast,2,...},...]} in context child_terminated
2015-11-01 04:42:30.588 [error] emulator Error in process <0.768.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 04:43:28.306 [error] emulator Error in process <0.776.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 04:45:52.268 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 04:45:59.632 [warning] <0.388.0> lager_error_logger_h dropped 57 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 04:45:59.632 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 04:45:59.717 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 04:46:10.401 [error] <0.651.0> gen_server ibot_nodes_srv_connector terminated with reason: {enoent,[{erlang,open_port,[{spawn_executable,"/home/alex/iBotOS/iBotOS/tp/new_projectdevfirst_cpp./first_cpp"},[{line,1000},stderr_to_stdout,{args,["/home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp.py","first_cpp","alex-N550JK","core@alex-N550JK","ibot_nodes_srv_connector","ibot_nodes_srv_topic","ibot_nodes_srv_service","ibot_nodes_srv_ui_interaction","ibot_events_srv_node_interaction",jv]}]],[]},{ibot_nodes_srv_connector,run_node,1,[{file,"src/ibot_nodes_srv_connector..."},...]},...]}
2015-11-01 04:46:10.402 [error] <0.651.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: {enoent,[{erlang,open_port,[{spawn_executable,"/home/alex/iBotOS/iBotOS/tp/new_projectdevfirst_cpp./first_cpp"},[{line,1000},stderr_to_stdout,{args,["/home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp.py","first_cpp","alex-N550JK","core@alex-N550JK","ibot_nodes_srv_connector","ibot_nodes_srv_topic","ibot_nodes_srv_service","ibot_nodes_srv_ui_interaction","ibot_events_srv_node_interaction",jv]}]],[]},{ibot_nodes_srv_connector,run_node,1,[{file,"src/ibot_nodes_srv_connector..."},...]},...]} in gen_server:terminate/7 line 804
2015-11-01 04:46:10.402 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.651.0> exit with reason {enoent,[{erlang,open_port,[{spawn_executable,"/home/alex/iBotOS/iBotOS/tp/new_projectdevfirst_cpp./first_cpp"},[{line,1000},stderr_to_stdout,{args,["/home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp.py","first_cpp","alex-N550JK","core@alex-N550JK","ibot_nodes_srv_connector","ibot_nodes_srv_topic","ibot_nodes_srv_service","ibot_nodes_srv_ui_interaction","ibot_events_srv_node_interaction",jv]}]],[]},{ibot_nodes_srv_connector,run_node,1,[{file,"src/ibot_nodes_srv_connector..."},...]},...]} in context child_terminated
2015-11-01 04:50:24.888 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 04:50:31.950 [warning] <0.388.0> lager_error_logger_h dropped 63 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 04:50:31.950 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 04:50:32.038 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 04:53:23.324 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 04:53:30.355 [warning] <0.388.0> lager_error_logger_h dropped 61 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 04:53:30.355 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 04:53:30.437 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 04:54:03.797 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 04:54:10.830 [warning] <0.388.0> lager_error_logger_h dropped 57 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 04:54:10.830 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 04:54:10.909 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 04:54:42.086 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 04:54:49.120 [warning] <0.388.0> lager_error_logger_h dropped 57 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 04:54:49.120 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 04:54:49.202 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 04:55:19.288 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 04:55:21.300 [warning] <0.388.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 04:55:21.301 [error] <0.637.0> Mnesia('core@alex-N550JK'): ** ERROR ** (core dumped to file: "/home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/MnesiaCore.core@alex-N550JK_1446_332121_299955")
 ** FATAL ** mnesia_tm crashed: {"Cannot read schema","/home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/Mnesia.core@alex-N550JK/schema.DAT",{error,{file_error,"/home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/Mnesia.core@alex-N550JK/schema.DAT",eacces}}} state: [<0.638.0>]

2015-11-01 04:55:26.281 [error] <0.632.0> gen_server ibot_nodes_srv_ui_interaction terminated with reason: bad record project_info in ibot_nodes_srv_ui_interaction:handle_cast/2 line 53
2015-11-01 04:55:26.281 [error] <0.632.0> CRASH REPORT Process ibot_nodes_srv_ui_interaction with 0 neighbours exited with reason: bad record project_info in ibot_nodes_srv_ui_interaction:handle_cast/2 line 53 in gen_server:terminate/7 line 804
2015-11-01 04:55:26.281 [error] <0.628.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_ui_interaction started with ibot_nodes_srv_ui_interaction:start_link() at <0.632.0> exit with reason bad record project_info in ibot_nodes_srv_ui_interaction:handle_cast/2 line 53 in context child_terminated
2015-11-01 04:55:31.300 [error] <0.639.0> gen_server mnesia_monitor terminated with reason: killed
2015-11-01 04:55:31.300 [error] <0.639.0> CRASH REPORT Process mnesia_monitor with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2015-11-01 04:55:31.300 [error] <0.642.0> gen_server mnesia_recover terminated with reason: killed
2015-11-01 04:55:31.300 [error] <0.642.0> CRASH REPORT Process mnesia_recover with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2015-11-01 04:55:31.300 [error] <0.637.0> Mnesia('core@alex-N550JK'): ** ERROR ** mnesia_event got unexpected event: {'EXIT',<0.640.0>,killed}

2015-11-01 04:55:31.300 [error] <0.637.0> CRASH REPORT Process mnesia_event with 0 neighbours exited with reason: killed in gen_event:terminate_server/4 line 302
2015-11-01 04:55:31.301 [error] <0.634.0> CRASH REPORT Process <0.634.0> with 0 neighbours exited with reason: {killed,{mnesia_sup,start,[normal,[]]}} in application_master:init/4 line 133
2015-11-01 04:55:31.301 [info] <0.389.0> Application mnesia exited with reason: {killed,{mnesia_sup,start,[normal,[]]}}
2015-11-01 04:56:34.622 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 04:56:41.620 [warning] <0.388.0> lager_error_logger_h dropped 57 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 04:56:41.620 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 04:56:41.856 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 05:00:13.476 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 05:00:20.521 [warning] <0.388.0> lager_error_logger_h dropped 60 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 05:00:20.521 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 05:00:20.704 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 05:00:44.375 [error] emulator Error in process <0.773.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 05:00:51.371 [error] emulator Error in process <0.781.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 05:02:35.478 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 05:02:42.543 [warning] <0.388.0> lager_error_logger_h dropped 64 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 05:02:42.543 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 05:02:42.702 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 05:27:32.102 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 05:27:39.166 [warning] <0.388.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 05:27:39.166 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 05:27:39.258 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 05:27:56.527 [error] <0.650.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:open_port({spawn_executable,"/bin/bash"}, [{line,1000},stderr_to_stdout,{args,["/home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/lib/ibot_cor...",...]}]) in ibot_core_func_cmd:execute_os_script_file/2 line 79
2015-11-01 05:27:56.527 [error] <0.650.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:open_port({spawn_executable,"/bin/bash"}, [{line,1000},stderr_to_stdout,{args,["/home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/lib/ibot_cor...",...]}]) in ibot_core_func_cmd:execute_os_script_file/2 line 79
2015-11-01 05:27:56.527 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.650.0> exit with reason bad argument in call to erlang:open_port({spawn_executable,"/bin/bash"}, [{line,1000},stderr_to_stdout,{args,["/home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/lib/ibot_cor...",...]}]) in ibot_core_func_cmd:execute_os_script_file/2 line 79 in context child_terminated
2015-11-01 05:28:48.626 [error] emulator Error in process <0.767.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 05:36:25.932 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 05:36:26.017 [warning] <0.388.0> lager_error_logger_h dropped 57 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 05:36:26.019 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 05:36:26.019 [info] <0.389.0> Application ibot_events started on node 'core@alex-N550JK'
2015-11-01 05:36:32.964 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 05:36:33.035 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 05:40:29.287 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 05:40:36.317 [warning] <0.388.0> lager_error_logger_h dropped 53 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 05:40:36.317 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 05:40:36.408 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 05:43:37.199 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 05:43:44.280 [warning] <0.388.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 05:43:44.280 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 05:43:44.420 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 05:43:54.483 [error] <0.651.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:open_port({spawn_executable,"/bin/bash"}, [{line,1000},stderr_to_stdout,{args,"/home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/lib/ibot_core-..."}]) in ibot_core_func_cmd:execute_os_script_file/1 line 79
2015-11-01 05:43:54.483 [error] <0.651.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:open_port({spawn_executable,"/bin/bash"}, [{line,1000},stderr_to_stdout,{args,"/home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/lib/ibot_core-..."}]) in ibot_core_func_cmd:execute_os_script_file/1 line 79
2015-11-01 05:43:54.483 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.651.0> exit with reason bad argument in call to erlang:open_port({spawn_executable,"/bin/bash"}, [{line,1000},stderr_to_stdout,{args,"/home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/lib/ibot_core-..."}]) in ibot_core_func_cmd:execute_os_script_file/1 line 79 in context child_terminated
2015-11-01 05:45:21.190 [error] <0.765.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:open_port({spawn_executable,"/bin/bash"}, [{line,1000},stderr_to_stdout,{args,"/home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/lib/ibot_core-..."}]) in ibot_core_func_cmd:execute_os_script_file/1 line 79
2015-11-01 05:45:21.190 [error] <0.765.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:open_port({spawn_executable,"/bin/bash"}, [{line,1000},stderr_to_stdout,{args,"/home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/lib/ibot_core-..."}]) in ibot_core_func_cmd:execute_os_script_file/1 line 79
2015-11-01 05:45:21.191 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.765.0> exit with reason bad argument in call to erlang:open_port({spawn_executable,"/bin/bash"}, [{line,1000},stderr_to_stdout,{args,"/home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/lib/ibot_core-..."}]) in ibot_core_func_cmd:execute_os_script_file/1 line 79 in context child_terminated
2015-11-01 05:50:44.475 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 05:50:51.744 [warning] <0.388.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 05:50:51.744 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 05:50:51.881 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 05:50:56.147 [error] <0.651.0> gen_server ibot_nodes_srv_connector terminated with reason: {system_limit,[{erlang,list_to_atom,["bash /home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/lib/ibot_core-1/priv/cpp_node_run.sh /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp first_cpp alex-N550JK core@alex-N550JK ibot_nodes_srv_connector ibot_nodes_srv_topic ibot_nodes_srv_service ibot_nodes_srv_ui_interaction ibot_events_srv_node_interaction jv"],[]},{ibot_core_func_cmd,exec,1,[{file,"src/ibot_core_func_cmd.erl"},{line,17}]},{ibot_nodes_srv_connector,handle_cast,2,[{file,...},...]},...]}
2015-11-01 05:50:56.147 [error] <0.651.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: {system_limit,[{erlang,list_to_atom,["bash /home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/lib/ibot_core-1/priv/cpp_node_run.sh /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp first_cpp alex-N550JK core@alex-N550JK ibot_nodes_srv_connector ibot_nodes_srv_topic ibot_nodes_srv_service ibot_nodes_srv_ui_interaction ibot_events_srv_node_interaction jv"],[]},{ibot_core_func_cmd,exec,1,[{file,"src/ibot_core_func_cmd.erl"},{line,17}]},{ibot_nodes_srv_connector,handle_cast,2,[{file,...},...]},...]} in gen_server:terminate/7 line 804
2015-11-01 05:50:56.148 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.651.0> exit with reason {system_limit,[{erlang,list_to_atom,["bash /home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/lib/ibot_core-1/priv/cpp_node_run.sh /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp first_cpp alex-N550JK core@alex-N550JK ibot_nodes_srv_connector ibot_nodes_srv_topic ibot_nodes_srv_service ibot_nodes_srv_ui_interaction ibot_events_srv_node_interaction jv"],[]},{ibot_core_func_cmd,exec,1,[{file,"src/ibot_core_func_cmd.erl"},{line,17}]},{ibot_nodes_srv_connector,handle_cast,2,[{file,...},...]},...]} in context child_terminated
2015-11-01 05:54:37.776 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 05:54:44.817 [warning] <0.388.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 05:54:44.817 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 05:54:44.917 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 05:57:03.314 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 05:57:10.343 [warning] <0.388.0> lager_error_logger_h dropped 56 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 05:57:10.343 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 05:57:10.425 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 05:57:21.356 [error] <0.651.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:open_port({spawn_executable,"/bin/bash"}, [{line,1000},stderr_to_stdout,{args,[["/home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/lib/ibot_c...",...]]}]) in ibot_core_func_cmd:execute_os_script_file/1 line 79
2015-11-01 05:57:21.356 [error] <0.651.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:open_port({spawn_executable,"/bin/bash"}, [{line,1000},stderr_to_stdout,{args,[["/home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/lib/ibot_c...",...]]}]) in ibot_core_func_cmd:execute_os_script_file/1 line 79
2015-11-01 05:57:21.356 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.651.0> exit with reason bad argument in call to erlang:open_port({spawn_executable,"/bin/bash"}, [{line,1000},stderr_to_stdout,{args,[["/home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/lib/ibot_c...",...]]}]) in ibot_core_func_cmd:execute_os_script_file/1 line 79 in context child_terminated
2015-11-01 06:01:05.134 [error] emulator Error in process <0.780.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 06:03:08.276 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 06:03:15.291 [warning] <0.388.0> lager_error_logger_h dropped 66 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 06:03:15.291 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 06:03:15.385 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 06:03:19.255 [error] <0.650.0> gen_server ibot_nodes_srv_connector terminated with reason: no function clause matching os:validate1(jv) line 366
2015-11-01 06:03:19.255 [error] <0.650.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: no function clause matching os:validate1(jv) line 366 in gen_server:terminate/7 line 804
2015-11-01 06:03:19.255 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.650.0> exit with reason no function clause matching os:validate1(jv) line 366 in context child_terminated
2015-11-01 06:04:44.574 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 06:04:51.604 [warning] <0.388.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 06:04:51.604 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 06:04:51.703 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 06:04:55.715 [error] <0.651.0> gen_server ibot_nodes_srv_connector terminated with reason: no function clause matching os:validate1(jv) line 366
2015-11-01 06:04:55.715 [error] <0.651.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: no function clause matching os:validate1(jv) line 366 in gen_server:terminate/7 line 804
2015-11-01 06:04:55.716 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.651.0> exit with reason no function clause matching os:validate1(jv) line 366 in context child_terminated
2015-11-01 06:05:45.979 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 06:05:46.064 [warning] <0.388.0> lager_error_logger_h dropped 57 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 06:05:46.065 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 06:05:46.066 [info] <0.389.0> Application ibot_events started on node 'core@alex-N550JK'
2015-11-01 06:05:53.011 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 06:05:53.096 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 06:05:58.344 [error] <0.651.0> gen_server ibot_nodes_srv_connector terminated with reason: no function clause matching os:validate1(jv) line 366
2015-11-01 06:05:58.344 [error] <0.651.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: no function clause matching os:validate1(jv) line 366 in gen_server:terminate/7 line 804
2015-11-01 06:05:58.344 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.651.0> exit with reason no function clause matching os:validate1(jv) line 366 in context child_terminated
2015-11-01 06:08:50.917 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 06:08:57.947 [warning] <0.388.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 06:08:57.947 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 06:08:58.033 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 06:09:20.767 [error] emulator Error in process <0.769.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 06:12:08.671 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 06:12:15.700 [warning] <0.388.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 06:12:15.700 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 06:12:15.783 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 06:12:36.114 [error] <0.651.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:12:36.114 [error] <0.651.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:12:36.114 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.651.0> exit with reason bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 06:13:27.031 [error] <0.775.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:13:27.031 [error] <0.775.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:13:27.032 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.775.0> exit with reason bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 06:15:06.016 [error] <0.790.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:15:06.016 [error] <0.790.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:15:06.017 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.790.0> exit with reason bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 06:15:11.036 [error] <0.805.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:15:11.037 [error] <0.805.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:15:11.037 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.805.0> exit with reason bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 06:15:42.295 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 06:15:49.325 [warning] <0.388.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 06:15:49.325 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 06:15:49.407 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 06:16:05.328 [error] emulator Error in process <0.771.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 06:16:17.058 [error] <0.650.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:16:17.058 [error] <0.650.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:16:17.058 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.650.0> exit with reason bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 06:16:31.356 [error] <0.776.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:16:31.356 [error] <0.776.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:16:31.356 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.776.0> exit with reason bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 06:16:32.949 [error] <0.778.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:16:32.949 [error] <0.778.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:16:32.949 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.778.0> exit with reason bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 06:16:33.920 [error] <0.780.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:16:33.920 [error] <0.780.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:16:33.921 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.780.0> exit with reason bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 06:16:34.710 [error] <0.782.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:16:34.711 [error] <0.782.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:16:34.711 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.782.0> exit with reason bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 06:16:35.527 [error] <0.784.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:16:35.527 [error] <0.784.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:16:35.527 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.784.0> exit with reason bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 06:16:37.828 [error] emulator Error in process <0.793.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 06:16:39.686 [error] <0.786.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:16:39.686 [error] <0.786.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:16:39.686 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.786.0> exit with reason bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 06:16:39.686 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.786.0> exit with reason reached_max_restart_intensity in context shutdown
2015-11-01 06:16:39.686 [info] <0.389.0> Application ibot_nodes exited with reason: shutdown
2015-11-01 06:16:56.802 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 06:17:03.882 [warning] <0.388.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 06:17:03.882 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 06:17:03.970 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 06:25:47.482 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 06:25:54.534 [warning] <0.388.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 06:25:54.534 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 06:25:54.614 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 06:26:29.371 [error] <0.650.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:26:29.371 [error] <0.650.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:26:29.371 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.650.0> exit with reason bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 06:28:00.374 [error] <0.774.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:28:00.375 [error] <0.774.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:28:00.375 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.774.0> exit with reason bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 06:29:33.673 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 06:29:40.712 [warning] <0.388.0> lager_error_logger_h dropped 63 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 06:29:40.712 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 06:29:40.807 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 06:29:50.722 [error] emulator Error in process <0.766.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 06:30:09.442 [error] emulator Error in process <0.773.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 06:30:16.819 [error] emulator Error in process <0.777.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 06:30:50.732 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 06:30:57.763 [warning] <0.388.0> lager_error_logger_h dropped 63 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 06:30:57.763 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 06:30:57.833 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 06:31:00.566 [error] emulator Error in process <0.767.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 06:33:29.740 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 06:33:36.778 [warning] <0.388.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 06:33:36.778 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 06:33:36.863 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 06:34:18.288 [error] emulator Error in process <0.769.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 06:37:31.643 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 06:37:38.689 [warning] <0.388.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 06:37:38.689 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 06:37:38.830 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 06:39:30.632 [error] emulator Error in process <0.769.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 06:42:46.846 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 06:42:53.908 [warning] <0.388.0> lager_error_logger_h dropped 60 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 06:42:53.908 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 06:42:53.990 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 06:46:24.023 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 06:46:31.081 [warning] <0.388.0> lager_error_logger_h dropped 60 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 06:46:31.081 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 06:46:31.164 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 06:46:42.781 [error] emulator Error in process <0.766.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 06:46:47.101 [error] emulator Error in process <0.769.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 06:49:42.933 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 06:49:43.016 [warning] <0.388.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 06:49:43.016 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 06:49:43.017 [info] <0.389.0> Application ibot_events started on node 'core@alex-N550JK'
2015-11-01 06:49:49.997 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 06:49:50.079 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 06:50:29.785 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 06:50:36.838 [warning] <0.388.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 06:50:36.838 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 06:50:36.924 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 06:50:50.760 [error] emulator Error in process <0.769.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 06:53:55.202 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 06:54:02.268 [warning] <0.388.0> lager_error_logger_h dropped 63 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 06:54:02.268 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 06:54:02.341 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 06:57:28.402 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 06:57:35.441 [warning] <0.388.0> lager_error_logger_h dropped 51 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 06:57:35.441 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 06:57:35.533 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 06:57:53.596 [error] <0.651.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("[sudo] password for alex: \e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nod...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:57:53.596 [error] <0.651.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("[sudo] password for alex: \e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nod...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:57:53.597 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.651.0> exit with reason bad argument in call to erlang:spawn("[sudo] password for alex: \e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nod...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 16:21:17.721 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 16:21:18.100 [warning] <0.388.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 16:21:18.101 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 16:21:18.101 [info] <0.389.0> Application ibot_events started on node 'core@alex-N550JK'
2015-11-01 16:21:24.801 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 16:21:24.974 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 16:22:12.382 [error] <0.651.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("[sudo] password for alex: \e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nod...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 16:22:12.382 [error] <0.651.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("[sudo] password for alex: \e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nod...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 16:22:12.382 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.651.0> exit with reason bad argument in call to erlang:spawn("[sudo] password for alex: \e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nod...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 16:22:26.838 [error] <0.773.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("[sudo] password for alex: \e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nod...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 16:22:26.838 [error] <0.773.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("[sudo] password for alex: \e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nod...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 16:22:26.838 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.773.0> exit with reason bad argument in call to erlang:spawn("[sudo] password for alex: \e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nod...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 16:22:56.302 [error] <0.798.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("[sudo] password for alex: \e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nod...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 16:22:56.302 [error] <0.798.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("[sudo] password for alex: \e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nod...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 16:22:56.302 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.798.0> exit with reason bad argument in call to erlang:spawn("[sudo] password for alex: \e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nod...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 16:28:40.436 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 16:28:47.489 [warning] <0.388.0> lager_error_logger_h dropped 63 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 16:28:47.489 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 16:28:47.580 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 16:29:00.044 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 16:29:07.098 [warning] <0.388.0> lager_error_logger_h dropped 60 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 16:29:07.099 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 16:29:07.194 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 16:29:10.211 [error] emulator Error in process <0.766.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 16:29:12.701 [error] emulator Error in process <0.769.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 16:31:33.644 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 16:31:40.684 [warning] <0.388.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 16:31:40.684 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 16:31:40.784 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 16:35:37.030 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 16:35:44.070 [warning] <0.388.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 16:35:44.070 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 16:35:44.156 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 16:37:06.598 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 16:37:13.630 [warning] <0.388.0> lager_error_logger_h dropped 66 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 16:37:13.630 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 16:37:13.735 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 16:40:41.640 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 16:40:48.689 [warning] <0.388.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 16:40:48.689 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 16:40:48.771 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 16:41:18.853 [error] emulator Error in process <0.774.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 16:44:23.308 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 16:44:30.379 [warning] <0.388.0> lager_error_logger_h dropped 64 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 16:44:30.379 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 16:44:30.459 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 16:46:10.217 [error] emulator Error in process <0.765.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 16:49:03.589 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 16:49:10.627 [warning] <0.388.0> lager_error_logger_h dropped 63 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 16:49:10.627 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 16:49:10.728 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 16:49:19.973 [error] <0.650.0> gen_server ibot_nodes_srv_connector terminated with reason: {enoent,[{erlang,open_port,[{spawn_executable,'echo 'alex!@#' | sudo -S /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp'},[{line,1000},stderr_to_stdout,{args,["first_cpp","alex-N550JK","core@alex-N550JK","ibot_nodes_srv_connector","ibot_nodes_srv_topic","ibot_nodes_srv_service","ibot_nodes_srv_ui_interaction","ibot_events_srv_node_interaction",jv]}]],[]},{ibot_nodes_srv_connector,run_node,1,[{file,"src/ibot_nodes_srv_connector.erl"},{line,254}]},{ibot_nodes_srv_connector,...},...]}
2015-11-01 16:49:19.974 [error] <0.650.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: {enoent,[{erlang,open_port,[{spawn_executable,'echo 'alex!@#' | sudo -S /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp'},[{line,1000},stderr_to_stdout,{args,["first_cpp","alex-N550JK","core@alex-N550JK","ibot_nodes_srv_connector","ibot_nodes_srv_topic","ibot_nodes_srv_service","ibot_nodes_srv_ui_interaction","ibot_events_srv_node_interaction",jv]}]],[]},{ibot_nodes_srv_connector,run_node,1,[{file,"src/ibot_nodes_srv_connector.erl"},{line,254}]},{ibot_nodes_srv_connector,...},...]} in gen_server:terminate/7 line 804
2015-11-01 16:49:19.974 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.650.0> exit with reason {enoent,[{erlang,open_port,[{spawn_executable,'echo 'alex!@#' | sudo -S /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp'},[{line,1000},stderr_to_stdout,{args,["first_cpp","alex-N550JK","core@alex-N550JK","ibot_nodes_srv_connector","ibot_nodes_srv_topic","ibot_nodes_srv_service","ibot_nodes_srv_ui_interaction","ibot_events_srv_node_interaction",jv]}]],[]},{ibot_nodes_srv_connector,run_node,1,[{file,"src/ibot_nodes_srv_connector.erl"},{line,254}]},{ibot_nodes_srv_connector,...},...]} in context child_terminated
2015-11-01 16:53:04.434 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 16:53:11.485 [warning] <0.388.0> lager_error_logger_h dropped 64 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 16:53:11.485 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 16:53:11.567 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 16:58:14.839 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 16:58:21.887 [warning] <0.388.0> lager_error_logger_h dropped 64 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 16:58:21.887 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 16:58:21.963 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 17:01:46.355 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 17:01:53.402 [warning] <0.388.0> lager_error_logger_h dropped 64 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 17:01:53.402 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 17:01:53.478 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
2015-11-01 17:06:35.404 [info] <0.389.0> Application lager started on node 'core@alex-N550JK'
2015-11-01 17:06:42.446 [warning] <0.388.0> lager_error_logger_h dropped 65 messages in the last second that exceeded the limit of 50 messages/sec
2015-11-01 17:06:42.446 [info] <0.389.0> Application mnesia exited with reason: stopped
2015-11-01 17:06:42.548 [info] <0.389.0> Application mnesia started on node 'core@alex-N550JK'
